src/main/java/browserview/ChromeDriverEx.java

```diff
@@ -50,9 +50,9 @@ public void get(String url, boolean isForceRefresh) throws WebDriverException {
             logger.info("Test loading page: " + url);
             testGet();
         } else {
-            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url) ||
+            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url)/* ||
                 driver.getCurrentUrl().equalsIgnoreCase(url + "/commits") ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/files"))) {
+                driver.getCurrentUrl().equalsIgnoreCase(url + "/files")*/)) {
```

Comment:
I'm still tracing the effects of this change, but all stable tests pass. I'll remove it in awhile